website/redirects.js

```diff
@@ -24,6 +24,22 @@ module.exports = [
     permanent: true,
   },
   */
+
+  // /s/* redirects for useful links that need a stable URL but we may need to
+  // change its destination in the future.
+  {
+    source: '/s/port-plan-failure',
+    destination:
```

Comment:
This is pointing at a redirected path. It should point to a relative developer path.

`/nomad/docs/operations/monitoring-nomad#progress`

Reply:
Unfortunately redirects with path fragments (`#`) don't work if they are relative, the fragment gets lost between hops. That's why we have to use absolute addresses here.

Reply:
I believe the issue with fragments should have been fixed earlier this week 👀